db/migrate/20200202044655_increment_reply_order.rb

```diff
@@ -0,0 +1,5 @@
+class IncrementReplyOrder < ActiveRecord::Migration[5.2]
+  def change
+    Reply.update_all('reply_order = reply_order + 1')
```

Comment:
I'm looking at this and a bit concerned that it might lock the table for a long time. I vaguely recall that we might've tested this - did we figure out that this ran quite quickly?

Reply:
Yeah, I made it a migration because it ran _so_ fast, even when I gave it a lot of replies. We'll probably need to maintenance briefly, but it shouldn't be very long.

Reply:
Ah, awesome!